bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/EntryLoggerAllocator.java

```diff
@@ -180,23 +180,24 @@ private synchronized BufferedLogChannel allocateNewLog(File dirForNextEntryLog,
         } while (testLogFile == null);
 
         File newLogFile = new File(dirForNextEntryLog, logFileName);
-        FileChannel channel = new RandomAccessFile(newLogFile, "rw").getChannel();
+        try (FileChannel channel = new RandomAccessFile(newLogFile, "rw").getChannel()) {
```

Comment:
is it a good idea to close the channel here? it is used by the logChannel that is being returned

Reply:
Reverted changes for this file